tests/reporting_utils.py

```diff
@@ -1,197 +1,143 @@
-
-    reportingWindow = testFixture.applySignature('ReportingWindow', market.getReportingWindow())
-
-    stakeTokenNo = testFixture.getOrCreateStakeToken(market, lastReportOutcomes)
-    stakeTokenYes = testFixture.getOrCreateStakeToken(market, firstReportDisputeOutcomes)
-
-    # If we buy the delta between outcome stakes that will be sufficient to make the outcome win
-    noStake = market.getPayoutDistributionHashStake(stakeTokenNo.getPayoutDistributionHash())
-    yesStake = market.getPayoutDistributionHashStake(stakeTokenYes.getPayoutDistributionHash())
-    stakeDelta = yesStake - noStake
-    stakeTokenNo.buy(stakeDelta + 1, sender=testFixture.testerKey[reporter])
-    tentativeWinner = market.getTentativeWinningPayoutDistributionHash()
-    assert tentativeWinner == stakeTokenNo.getPayoutDistributionHash()
-
-    # To progress into the LAST DISPUTE phase we move time forward
-    testFixture.contracts["Time"].setTimestamp(reportingWindow.getDisputeStartTime() + 1)
-    assert market.getReportingState() == testFixture.contracts['Constants'].LAST_DISPUTE()
-
-    logs = []
-    captureFilteredLogs(testFixture.chain.head_state, testFixture.contracts['Augur'], logs)
-
-    # Making a dispute at this phase will progress the market into FORKING
-    assert market.disputeLastReporters(sender=tester.k0)
-    assert market.getReportingState() == testFixture.contracts['Constants'].FORKING()
-
-    # Confirm the last dispute logging and universe fork logging works
-    assert len(logs) >= 3
-    log1 = logs[1]
-    log2 = logs[2] if len(logs) == 3 else logs[3]
-    assert log1['_event_type'] == 'UniverseForked'
-    assert log1['universe'] == universe.address
-
-    assert log2['_event_type'] == 'ReportsDisputed'
-    assert log2['reportingPhase'] == testFixture.contracts['Constants'].LAST_DISPUTE()
-    assert log2['disputer'] == bytesToHexString(tester.a0)
-    assert log2['disputeBondAmount'] == testFixture.contracts['Constants'].LAST_REPORTERS_DISPUTE_BOND_AMOUNT()
-    assert log2['market'] == market.address
-
-def finalizeForkingMarket(reportingFixture, universe, market, finalizeByMigration, yesMigratorAddress, yesMigratorKey, noMigratorAddress1, noMigratorKey1, noMigratorAddress2, noMigratorKey2, firstReportOutcomes, secondReportOutcomes):
-    reputationToken = reportingFixture.applySignature('ReputationToken', universe.getReputationToken())
+        feeWindow = fixture.applySignature('FeeWindow', market.getFeeWindow())
+        fixture.contracts["Time"].setTimestamp(feeWindow.getStartTime() + 1)
+        # This will also use the InitialReporter which is not a DisputeCrowdsourcer, but has the called function from abstract inheritance
+        winningReport = fixture.applySignature('DisputeCrowdsourcer', market.getWinningReportingParticipant())
+        winningPayoutHash = winningReport.getPayoutDistributionHash()
+        firstReportWinning = market.derivePayoutDistributionHash(payoutNumerators, False) == winningPayoutHash
```

Comment:
Minor test performance optimization: Recommend moving this into the else block as the result is only used in that code path.